### PR TITLE
Optimize some wrong usage and spelling

### PR DESCRIPTION
**Pull request by wgliang**
**- What I did**

1.The word is misspelled

2.should use bytes.Equal() instead of bytes.Compare()

3.should use time.Since instead of time.Now().Sub

4.should replace loop with args = append(args, params...)

5._ = <-ch can be simplified to <-ch

**- How I did it**
NONE
**- How to verify it**
NONE
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]
LGTM

File: integration-cli/docker_api_containers_test.go [hunk 1]
```diff
@@ -1027,8 +1027,8 @@ func (s *DockerSuite) TestContainerAPIWait(c *check.C) {
 	select {
 	case err = <-errC:
 		c.Assert(err, checker.IsNil)
-	case waitres := <-waitresC:
-		c.Assert(waitres.StatusCode, checker.Equals, int64(0))
+	case waiters := <-waitresC:
```
Comment (AkihiroSuda) [inline integration-cli/docker_api_containers_test.go]: No, "waitres" (wait result) is correct

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]
Also, please update the title of the PR

File: integration-cli/docker_api_containers_test.go (hunk 1 above)
Comment (wgliang) [reply integration-cli/docker_api_containers_test.go]: Get！

**Comment by wgliang** [comment]
I've updated it.

**Review by boaz0 — APPROVED** [review]
LGTM! GJ

**Review by dnephin — APPROVED** [review]
LGTM

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Comment by ripcurld0** [comment]
@wgliang can you update the commit title to be the same as the title of the PR?

Thanks.

**Comment by wgliang** [comment]
@ripcurld0 Ok,I've updated it.

**Review by lowenna — APPROVED** [review]
LGTM

**Pull request merged**